<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputFormat</class>
 <widget class="QDialog" name="OutputFormat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Output Format</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Output format:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditOutputFormat">
       <property name="placeholderText">
        <string>$short_link$</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="help">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Name: $name$  ; example.txt
File Path: $path$  ; /home/example.txt
Size: $size$  ; 1.22 MB
Short Link: $short_link$  ; https://anonfile.com/p0v3ycF2n5
Full Link: $full_link$  ; https://anonfile.com/p0v3ycF2n5/example.txt
Error: $error$  ; Error, invalid reply. Verify your Internet connection
Example:
$name$: $short_link$ =&gt; example.txt: https://anonfile.com/p0v3ycF2n5
</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="output_format.qrc"/>
 </resources>
 <connections/>
</ui>
